tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_types.ts

```diff
@@ -28,6 +28,31 @@ export abstract class TBFeatureFlagDataSource {
    * not have enough information to provide values.
    */
   abstract getFeatures(enableMediaQuery?: boolean): Partial<FeatureFlags>;
+
+  /**
+   * Stores the given feature flag values in localStorage to allow for more
+   * persistent flag state.
+   *
+   * @param flags An object holding the feature flags that are to be stored.
+   */
+  abstract persistFeatureFlags(flags: Partial<FeatureFlags>): void;
+
+  /**
+   * Removes the local storage override of the given flag. If the flag is not
```

Comment:
Nit: elsewhere you use 'localStorage' but here you use 'local storage'.

Reply:
Good catch.